winpr/CMakeLists.txt

```diff
@@ -227,6 +227,17 @@ set(OPENSSL_FEATURE_TYPE "RECOMMENDED")
 		set(TESTING_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/Testing")
 endif()
 
+if (NOT WIN32)
```

Comment:
shouldn´t this move to `WinPR` as this module is there and should be buildable without `FreeRDP`?

Reply:
Not sure to understand your comment, I've put that in `winPR` precisely because I wanted FreeRDP to not care about `libpkcs11` and have winpr hiding this.

Reply:
ok, strange, did show under `CMakeLists.txt` instead of `winpr/CMakeLists.txt` last time? anyway, here it is perfekt.